examples/do-not-use/main.tf

```diff
@@ -789,19 +789,41 @@ locals {
 
 data "aws_iam_policy_document" "aws_load_balancer_controller" {
```

Comment:
this is the policy we should be following https://raw.githubusercontent.com/kubernetes-sigs/aws-load-balancer-controller/v2.5.1/docs/install/iam_policy.json

Note: partitions and DNS suffixes will need to match the regions

Reply:
This policy triggers a lot of tfsec issues. The one that I added is the one on this [PR](https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/1398), that was not merged in the past. 

I fixed all the partitions, regions, and account IDs when fixed the policy. Didn't find other DNS suffixes matches other than line 800.